sources/packages/web/src/components/fa-application/ConfirmStudentAidBCProfileInfo.vue

```diff
@@ -71,16 +70,17 @@ import { StudentService } from "../../services/StudentService";
 export default {
   setup() {
     //Creating a reactive array
-    const studentConfirmInfo = ref([]);
+    const studentInfo = ref([]);
+    const studentContact = ref([]);
```

Comment:
Do we need a second object to hold the contact information?
Can we use just studentInfo?

Reply:
i tried it , and nested infromation was being a problem

Reply:
And to your comment on addressline2 and space, i have fixed it but that's not really related to this user story. So if there are more requirements around formatting of address(which i don't think there will be) we can tackle it later